packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -94,25 +87,39 @@ const ShowHide = (props) => {
 
   const theme = React.useContext(ThemeContext);
   const contentRef = React.useRef(null);
+  const [activeElement, setActiveElement] = React.useState(null);
+  const focusRefClassName = 'show-hide-ref-focusable';
 
   /**
-   * Upon showing hidden content, set focus to the child element if focusRef provided, othervise set focus to the content container.
-   * That would allow assistive technologies to read the newly revealed content.
+   * Upon showing hidden content, set activeElement to focusRef element if provided, othervise set it to content container.
    */
   React.useEffect(() => {
+    let newActiveElement = null;
     if (isOpen) {
-      let element = null;
       if (focusRef?.current) {
-        element = focusRef?.current;
+        if (checkIfAddTabindex(focusRef?.current)) {
+          // not focusable element with no tabindex set, safe to temporary add tabindex that will later be removed on blur
+          focusRef?.current.setAttribute('tabIndex', '-1');
+          focusRef?.current.setAttribute('role', 'group');
+          focusRef?.current.classList.add(focusRefClassName); // needed for onBlur cleanup
+        }
+        newActiveElement = focusRef?.current;
```

Comment:
You would only set the state for the contentRef.  There is no need to set if we are using the focusRef.

Reply:
No, I need to know the actual active element being contentRef, focusRef, or null. Here is the explanation: https://github.com/cerner/terra-core/pull/3838#discussion_r1263961154

Reply:
This also applies to that: https://github.com/cerner/terra-core/pull/3838#discussion_r1264094548